app/components/task-selection/component.js

```diff
@@ -128,25 +127,25 @@ export default class TaskSelectionComponent extends Component {
    * @property {Customer} _customer
    * @private
    */
-  @tracked
-  _customer = null;
+  @localCopy("args.initial.customer")
+  _customer = this.args.initial.customer || null;
 
   /**
    * The manually selected project
    *
    * @property {Project} _project
    * @private
    */
-  @tracked
-  _project = null;
+  @localCopy("args.initial.project")
+  _project = this.args.initial.project || null;
 
   /**
    * The manually selected task
    *
    * @property {Task} _task
    * @private
    */
-  @tracked
+  @localCopy("args.initial.task")
   _task = null;
```

Comment:
Move those above the constructor, so it's more obvious that this is locale state.